kolibri/core/assets/src/views/TextTruncator.vue

```diff
@@ -135,6 +135,9 @@
   .truncated {
     overflow: hidden;
     text-overflow: ellipsis;
+    a {
+      text-decoration: none !important;
+    }
```

Comment:
It looks like this is overriding styling coming from inside `shave`? It could be useful to add a comment describing what the intended outcome is, e.g.:

```suggestion
    // this style ensures that ____
    a {
      text-decoration: none !important;
    }
```

Reply:
Added comment!